util/monoclock.h

```diff
@@ -11,4 +11,12 @@
  */
 int monoclock_get(struct timeval *);
 
+/**
+ * monoclock_get_cputime(tv):
+ * Store how much time the process has been running in ${tv} if
```

Comment:
"how much time the process has been running in ${tv}" seems dangerously easy to misparse.  Maybe "Store in ${tv} the duration the process has been running..."?

Reply:
That's easier to parse, but it looks really silly for the simple `monoclock_get()`: "Store in ${tv} the current time.".  And it also looks weird to have different grammar in such simple functions.

I've uploaded this change so you can see how it looks in reality.  Despite the awkwardness, I actually *do* prefer the new version, since it's easy to parse.